tests/ios/xcodeproj/build.sh

```diff
@@ -1,13 +1,30 @@
 set -eux
+# DESTINATION_TYPE should be one of 'simulator' or 'device'
+# this is used to pick the correct sdk when invoking 'xcodebuild' in this script
+DESTINATION_TYPE=${1:-simulator}
```

Comment:
what does `${1:-simulator}` do again? cos the check for either simulator or device is below, why this one needs to have `simulator` inside... i am bash newbie still 😢

Reply:
Same! Didn't know about this